modules/@apostrophecms/i18n/ui/apos/components/AposI18nLocalize.vue

```diff
@@ -587,7 +588,12 @@ export default {
           }
         }
       }
-      this.close();
+
+      // Prevent flashing of the UI if the request returns quickly
+      setTimeout(() => {
```

Comment:
Without this when the docs are localized quickly, the screen "flashes" and you're not able to see the busy screen. While it's nice we're super fast, it's not so nice that there's a blip after clicking and before being directed back to the page.